src/features/collapsible-sidebar/__tests__/__snapshots__/CollapsibleSidebarLogo.test.js.snap

```diff
@@ -246,67 +246,80 @@ exports[`components/core/collapsible-sidebar/CollapsibleSidebar render 2`] = `
         }
       >
         <Tooltip
-          constrainToScrollParent={false}
-          constrainToWindow={true}
           isDisabled={true}
           isTabbable={false}
           position="middle-right"
-          theme="default"
         >
-          <CollapsibleSidebarLogo__StyledToggleButton
-            aria-label="Expand"
-            className="bdl-CollapsibleSidebar-toggleButton"
-            type="button"
+          <Tooltip
```

Comment:
Why does this render two `Tooltips`?

Reply:
This snapshot is generated with `mount` instead of `shallow`. Same thing happened with the [SidebarToggleButton snapshot test](https://github.com/box/box-ui-elements/blob/master/src/components/sidebar-toggle-button/__tests__/SidebarToggleButton.test.js#L7)